apps/server/src/modules/management/seed-data/roles.ts

```diff
@@ -0,0 +1,465 @@
+// All user accounts are organized by school in a single array
+
+import { IRoleProperties, Permission, Role, RoleName } from '@shared/domain';
+import { roleFactory } from '@shared/testing';
+import { DeepPartial } from 'fishery';
+
+type SeedRoleProperties = Omit<IRoleProperties, 'roles'> & {
+	id: string;
+	createdAt: string;
+	updatedAt: string;
+	roles: RoleName[];
+};
+
+const roleSeedData: { [key: string | RoleName]: SeedRoleProperties } = {
+	user: {
+		id: '0000d186816abba584714c95',
+		createdAt: '2017-01-01T00:06:37.148Z',
+		updatedAt: '2023-04-04T12:33:09.676Z',
+		name: RoleName.USER,
+		roles: [],
+		permissions: [
+			Permission.ACCOUNT_EDIT,
```

Comment:
Can you change the https://github.com/hpi-schul-cloud/schulcloud-server/blob/main/apps/server/src/shared/testing/user-role-permissions.ts#L7 to use the Permission enum and use the result in this file too?

I want to avoid that we have 2 sources, that must be maintained. Also for the non user roles like team.....
The course....